CMakeLists.txt

```diff
@@ -473,3 +473,43 @@ ExternalProject_Add(aws-sdk-cpp
     -DCMAKE_INSTALL_PREFIX:PATH=${TRITON_THIRD_PARTY_INSTALL_PREFIX}/aws-sdk-cpp
   PATCH_COMMAND python3 ${CMAKE_CURRENT_SOURCE_DIR}/tools/install_src.py --src <SOURCE_DIR> ${INSTALL_SRC_DEST_ARG}
 )
+
+#
+# Build OpenTelemetry C++ library
+# [FIXME] Remove WIN flag when windows build works
```

Comment:
Please provide a JIRA ticket number for this future work and a link to the documentation about Windows build not working. There should be examples in the repos.